gradle/dependencies.gradle

```diff
@@ -77,7 +76,6 @@ ext {
       mapboxSdkTurf             : "com.mapbox.mapboxsdk:mapbox-sdk-turf:${version.mapboxSdkServices}",
       mapboxSdkDirectionsModels : "com.mapbox.mapboxsdk:mapbox-sdk-directions-models:${version.mapboxSdkServices}",
       mapboxSdkRefreshModels    : "com.mapbox.mapboxsdk:mapbox-sdk-directions-refresh-models:${version.mapboxSdkServices}",
-      mapboxEvents              : "com.mapbox.mapboxsdk:mapbox-android-telemetry:${version.mapboxEvents}",
       mapboxCore                : "com.mapbox.mapboxsdk:mapbox-android-core:${version.mapboxCore}",
```

Comment:
looks like we can, let me check

Reply:
we cannot, following classes are part of mapbox-android-core:
- `com.mapbox.android.core.location.LocationEngine`;
- `com.mapbox.android.core.location.LocationEngineRequest`

Reply:
Weren't we planning to reintroduce this same classes under the same packages but directly from the Common SDK @tarigo @tatiana-yan?

In the current situation, we'd still need to maintain https://github.com/mapbox/mapbox-events-android/.

Reply:
we have `com.mapbox.common.location.compat.LocationEngine`, `com.mapbox.common.location.compat.LocationEngineRequest`

Reply:
@tatiana-yan we cannot migrate to `com.mapbox.common.location.compat.LocationEngine` and `com.mapbox.common.location.compat.LocationEngineRequest`, because changing a package name is the breaking changes.

Reply:
Refs https://github.com/mapbox/mapbox-sdk-common/pull/1937#discussion_r887841808. @tarigo was this eventually addressed in some way?

Reply:
I see https://github.com/mapbox/mapbox-sdk-common/issues/2807 is still open (I can't find corresponding JIRA ticket).

Reply:
This means that Nav SDK cannot drop the dependency on MME (at least the location part) because `LocationEngine` and `LocationEngineRequest` are part of our public API.

Reply:
Let's discuss this separately, it doesn't block the PR.

Reply:
For the external customers there is a plan to replace it by renaming Common's [LocationEngine compat](https://github.com/mapbox/mapbox-sdk-common/tree/main/components/location/platform/android/src/main/java/com/mapbox/common/location/compat), you can keep core dependency because it's a public API but migrate to compat -  it would be great to test it. 
Maps  SDK has already [adopted it](https://github.com/mapbox/mapbox-maps-android/pull/1672/files).